packages/pds/src/repo/prepare.ts

```diff
@@ -301,6 +302,14 @@ function assertNoExplicitSlurs(rkey: string, record: RepoRecord) {
     toCheck += ' ' + record.displayName
   } else if (isPost(record)) {
     toCheck += record.tags?.join(' ')
+
```

Comment:
Not related to your work, but just noticed there may be a tiiiny baby bug on the line above this one: I expect it will append the string `undefined` when the post has no tags.

Reply:
Ah, but that was my _previous_ work. Fixed!